.github/workflows/doc-build.yml

```diff
@@ -100,6 +100,14 @@ jobs:
         git clean -f -d
         git log --graph --oneline HEAD...${PR_HEAD}
 
+    - name: Set up Python
+      if: github.event_name == 'pull_request'
```

Comment:
Why this `if`? The installation step does not have that.

Reply:
You are right, fixed.